server/sonar-web/src/main/js/apps/users/components/UsersSelectSearch.js

```diff
@@ -80,12 +81,15 @@ export default class UsersSelectSearch extends React.PureComponent {
     this.mounted = false;
   }
 
-  filterSearchResult = ({ users } /*: { users: Array<Option> } */) =>
-    users.filter(user => !this.props.excludedUsers.includes(user.login)).slice(0, LIST_SIZE);
+  filterSearchResult = (users /*: Array<Option> */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)